Cubical/HITs/Truncation/Properties.agda

```diff
@@ -204,3 +217,174 @@ groupoidTrunc≃Trunc1 =
+                                                     {B = λ x → encode-fun ∣ a ∣ ∣ b ∣ (decode-fun ∣ a ∣ ∣ b ∣ x) ≡ x}
+                                                     (λ x → (sym ((snd (isOfHLevel∥∥ {A = a ≡ b} neg2))
+                                                                  (encode-fun ∣ a ∣ ∣ b ∣ (decode-fun ∣ a ∣ ∣ b ∣ x)))
+                                                               ∙ ((snd (isOfHLevel∥∥ {A = a ≡ b} neg2)) x))
+                                                            ,
+                                                            λ y  → hLevelSuc (2+ (suc neg2)) (∥ a ≡ b ∥ neg2)
+                                                                     (hLevelSuc (2+ neg2) (∥ a ≡ b ∥ neg2) (isOfHLevel∥∥ {A = a ≡ b} (neg2)))
+                                                                     (encode-fun ∣ a ∣ ∣ b ∣ (decode-fun ∣ a ∣ ∣ b ∣ x)) x
+                                                                      ((sym ((snd (isOfHLevel∥∥ {A = a ≡ b} neg2))
+                                                                             (encode-fun ∣ a ∣ ∣ b ∣ (decode-fun ∣ a ∣ ∣ b ∣ x)))
+                                                                          ∙ ((snd (isOfHLevel∥∥ {A = a ≡ b} neg2)) x)))
+                                                                          y)
+                                                     (J {ℓ}{B}{a}{ℓ}
+                                                     ((λ y p → (encode-fun {n = neg2}) ∣ a ∣ ∣ y ∣
+                                                                           ((decode-fun ∣ a ∣ ∣ y ∣) ∣ p ∣) ≡ ∣ p ∣))
+                                                     (enc-refl {n = neg2} ∣ a ∣ )
+                                                     {b})
+          helper {ℓ = ℓ} {B = B} {n = suc n} a b = ind {A = (a ≡ b)}
+                                                      {n =  suc n}
+                                                      {B = λ x → encode-fun ∣ a ∣ ∣ b ∣ (decode-fun ∣ a ∣ ∣ b ∣ x) ≡ x}
+                                                      (λ x → (hLevelP {n = suc n} ∣ a ∣ ∣ b ∣ (encode-fun ∣ a ∣ ∣ b ∣ (decode-fun ∣ a ∣ ∣ b ∣ x)) x) )
+                                                      (J {ℓ}{B}{a}{ℓ} ((λ y p → (encode-fun {n = suc n}) ∣ a ∣ ∣ y ∣ ((decode-fun ∣ a ∣ ∣ y ∣) ∣ p ∣) ≡ ∣ p ∣))
+                                                      (enc-refl {n = suc n} ∣ a ∣ )
+                                                      {b})
+
+        {- The final Iso established -}
+        IsoFinal : ∀ {ℓ} {B : Type ℓ} {n : ℕ₋₂} (x y : ∥ B ∥ (suc n)) → Iso (x ≡ y) (P x y)
+        IsoFinal x y = iso (encode-fun x y ) (decode-fun x y) (P-linv x y) (P-rinv x y)
+
+IsoIdTrunc : {a b : A} (n : ℕ₋₂) → Iso (_≡_ {A = ∥ A ∥ (suc n)} ∣ a ∣ ∣ b ∣) (∥ a ≡ b ∥ n)
+IsoIdTrunc {a = a} {b = b} n = IsoFinal {n = n} ∣ a ∣ ∣ b ∣
+
+IsoΩ : {a : A} (n : ℕ₋₂) → Iso (_≡_ {A = ∥ A ∥ (suc n)} ∣ a ∣ ∣ a ∣) (∥ a ≡ a ∥ n)
```

Comment:
Why state these as Iso and not Equiv or Path? I think it would be useful to have the other formulations as well (especially Path)